src/pages/login.tsx

```diff
@@ -1,48 +1,126 @@
 import { NextPage } from "next";
 import Image from "next/image";
-import Logo from "../public/TUC-farbig.png";
+import Logo from "../public/TUC-einfarbig.png";
+import { useState } from "react";
+import { useRouter } from "next/router";
+import useRedirectIfLoggedIn from "../hooks/useRedirectIfLoggedIn";
+
+const LoginPage: NextPage = () => {
+  useRedirectIfLoggedIn();
```

Comment:
wenn man schon eingeloggt ist, führt das dazu, dass der next parameter ignoriert wird. finde ich ein wenig unschön

Reply:
Erledigt in [c0b2aff](https://github.com/top-22/forum/pull/14/commits/c0b2affccb6913e7a0a7eb15c182bc5b0100fde6)